lib/assets/tree/main.css

```diff
@@ -0,0 +1,38 @@
+.app {
+  font-family: "JetBrains Mono", monospace;
+  font-size: 0.875rem;
+}
+
+div.item {
+  display: flex;
+}
+
+div.clickable {
+  cursor: pointer;
+}
+
+.ri {
+  font-size: 1.25rem;
+  vertical-align: middle;
+}
+
+.ri-empty {
+  display: inline-block;
+  width: 1.25rem;
+}
+
+ol {
+  display: none;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  padding-left: 2rem;
```

Comment:
Ideally we would make the indentation consistent with arrow-less inspect. Here we can use `2ch`, which corresponds to the width of two spaces.

For the arrow we can't use `2ch` directly because it uses the icon font, but I we can wrap the icon in another element, make that element `2ch` wide and then maybe align the icon if needed.